src/Components/Calendar/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Calendar } from 'react-native-toggle-calendar';
+import moment from 'moment';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import CalendarHeader from './CalendarHeader';
+let calendarDate = moment();
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  calendarStyle: {
+    paddingLeft: 0,
+    paddingRight: 0,
+  },
+  firstHorizontalLine: {
+    marginTop: 5,
+    borderBottomColor: '#e8f1fb',
+    borderBottomWidth: 3,
+  },
+  iconContainer: {
+    flexDirection: 'row',
+    marginTop: 10,
+    marginBottom: 10,
+    justifyContent: 'space-around',
+  },
+  firstIconContainer: {
+    flexDirection: 'row',
+  },
+  ovulationText: {
+    alignSelf: 'center',
+    marginLeft: 5,
+  },
+  secondIconContainer: {
+    flexDirection: 'row',
+  },
+  periodsText: {
+    alignSelf: 'center',
+    marginLeft: 5,
+  },
+});
+
+const CalendarComponent = props => {
+  const [calendarState, setCalendarDate] = React.useState(calendarDate.format('YYYY-MM-DD'));
+  function onPressArrowLeft() {
+    calendarDate = calendarDate.add(-1, 'month');
+    updateCalendarDate();
+  }
+  function onPressArrowRight() {
+    calendarDate = calendarDate.add(1, 'month');
+    updateCalendarDate();
+  }
+  function onPressListView() {
+    //his.setState({ horizontal: true });
+  }
+  function onPressGridView() {
+    //this.setState({ horizontal: false });
+  }
+  function onDayPress(date) {
+    // calendarDate = moment(date.dateString);
+    // this.updateCalendarDate();
+  }
+  function updateCalendarDate() {
+    setCalendarDate(calendarDate.format('YYYY-MM-DD'));
+  }
+  return (
+    <View style={styles.container}>
+      <Calendar
+        current={calendarState}
+        // dayComponent={CalendarDayComponent}
+        calendarHeaderComponent={CalendarHeader}
+        headerData={{
+          calendarDate: calendarDate.format('MMM, YYYY'),
+        }}
+        style={styles.calendarStyle}
+        theme={{
+          textDayFontSize: 18,
+        }}
+        markingType={'custom'} //{"period"}
+        markedDates={{
+          // [this.state.calendarDate]: {
```

Comment:
Remove the commented code

Reply:
Some methods do not have a body but it was a must to define them. And day component may be needed in future. For that purpose I let it remain commented.